.github/workflows/deploy.yml

```diff
@@ -107,11 +108,6 @@ jobs:
           path: guides/build/
 
   build-pdf:
-    if: >
-      github.repository_owner == 'theforeman' &&
-      (github.ref == 'refs/heads/master' ||
-      startsWith(github.ref, 'refs/heads/2.') ||
-      startsWith(github.ref, 'refs/heads/3.'))
```

Comment:
I moved this down, we now build one PDF version for PR CI and all for deploy commits.